astropy/io/fits/hdu/table.py

```diff
@@ -616,7 +616,7 @@ def _update_column_attribute_changed(self, column, col_idx, attr,
             else:
```

Comment:
Is it just me, or does this `else` does not have an accompanying `if`? I thought indentation matter in Python.